src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx

```diff
@@ -0,0 +1,97 @@
+* **Sync discussions with Slack.**  With our Slack integration you can share links, screenshots, and comments to Slack in seconds. All teammates  can reply from Slack and discussions stay in sync, no matter where teammates contribute. [Learn more about the Slack integration](https://docs-preview.newrelic.com/docs/collaboration-slack-integration).
+* **Create shared knowledge and history.** Collaborating within the New Relic platform will allow your team to build a searchable body of shared knowledge.
+## Comment on any page [#comment-on-any-page]
+
+On the global navigation, there's a new “add comment” icon on most pages. A green dot on the icon means there is actually an open discussion on the page. A gray dot means there are closed discussions that can perhaps answer new questions or help resolve current issues. 
+
+<img
+  title="Comment on any page "
+  alt="A screenshot of the UI that depicts commenting on a specific page."
+  src={green_dot}
+/>
+
+If you want to ask a question, start a discussion, or share a permalink or screenshot to Slack, click the icon to open the comment panel. Type your comment and start a discussion on the page you're looking at. When you comment on a dashboard or an error in New Relic, that comment stays with your data. You don't need to spend time repeating information that can be seen at a glance, and everybody knows where to find an active, solution-oriented conversation.
+
+You may notice the “add comment” icon is disabled on some pages. By design, in a multi-account organization you can't discuss pages that either don't have data associated with a specific account. You also won't be able to discuss on pages that present data belonging to multiple accounts in a single view. Simply selecting a specific account or clicking through to a specific item or entity will usually show the “add comment” icon.
+
+## At-mention your teammates and notify by email [#at-mention-your-teammates]
+
+Like other collaboration tools, when you at-mention (@) your teammates in a comment, they'll be notified by email. If you at-mention a teammate that isn't part of the preview or isn't using the new redesign, they won't have access to the discussion on New Relic. 
+
+## Share links and screenshots with Slack [#sync-with-slack]
+
+Whenever you start a new discussion, you can sync it with Slack so you can collaborate with all your teammates, even those that aren't on New Relic.
+
+Once the Slack app is configured and you are authenticated, select the Slack channel where you want your discussion synced. If you regularly use the same Slack channel it'll be prefilled so no extra clicks needed. From Slack, any teammate, even if they are not a New Relic user, can reply and their comment will be brought back into the thread on New Relic. Likewise subsequent replies added to the thread on New Relic will also be added to the thread on Slack.
```

Comment:
For the second sentence I think there should be a comma after "Slack channel". Also in this sentence, there should be an "are" between "extra" and "clicks"

Reply:
There are a lot of commas in the third sentence that can make it kind of hard to parse. What do you think about something like "Any teammate can reply in Slack, and their comment will be integrated into the thread on New Relic even if they are not a New Relic user." or something along those lines?

Reply:
I think you can cut Likewise from the last sentence.